netbox_acls/migrations/0005_acl_rule_source_and_destination_objects.py

```diff
@@ -0,0 +1,253 @@
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def copy_prefix_assignments(apps, schema_editor):
+    """
+    Copy Source and Destination Prefix ForeignKey IDs to the GenericForeignKey
+    fields.
+    """
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    Prefix = apps.get_model("ipam", "Prefix")
+    ACLStandardRule = apps.get_model("netbox_acls", "ACLStandardRule")
+    ACLExtendedRule = apps.get_model("netbox_acls", "ACLExtendedRule")
+
+    ACLStandardRule.objects.filter(_source_prefix__isnull=False).update(
+        source_type=ContentType.objects.get_for_model(Prefix),
+        source_id=models.F("_source_prefix_id"),
+    )
+    ACLExtendedRule.objects.filter(_source_prefix__isnull=False).filter(_destination_prefix__isnull=False).update(
+        source_type=ContentType.objects.get_for_model(Prefix),
+        source_id=models.F("_source_prefix_id"),
+        destination_type=ContentType.objects.get_for_model(Prefix),
+        destination_id=models.F("_destination_prefix_id"),
+    )
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("extras", "0129_fix_script_paths"),
+        ("ipam", "0082_add_prefix_network_containment_indexes"),
```

Comment:
This migration was introduced in NetBox v4.3.6, which was released on https://github.com/netbox-community/netbox/commit/8bb47dad0f9f0d6d0b11bcf685a0c47f47233fdc. See my previous comment for more information on why this might be a problem.